docs/src/components/ExternalExampleLayout.tsx

```diff
@@ -11,38 +11,41 @@ import {
 } from 'docs/src/utils'
 import PageNotFound from '../views/PageNotFound'
 import { babelConfig, importResolver } from './Playground/renderConfig'
+import Provider from '../../../src/components/Provider/Provider'
 
 const examplePaths = exampleSourcesContext.keys()
 
 const ExternalExampleLayout: any = props => {
-  const { exampleName } = props.match.params
+  const { exampleName, rtl } = props.match.params
   const exampleFilename = exampleKebabNameToSourceFilename(exampleName)
-
+  console.log(exampleFilename)
   const examplePath = _.find(examplePaths, path => {
     const { exampleName } = parseExamplePath(path)
     return exampleFilename === exampleName
   })
-
+  console.log(examplePath)
   if (!examplePath) return <PageNotFound />
   const exampleSource: ExampleSource = exampleSourcesContext(examplePath)
 
   return (
-    <SourceRender
-      babelConfig={babelConfig}
-      source={exampleSource.js}
-      renderHtml={false}
-      resolver={importResolver}
-    >
-      <SourceRender.Consumer>
-        {({ element, error }) => (
-          <>
-            {element}
-            {/* This block allows to see issues with examples as visual regressions. */}
-            {error && <div style={{ fontSize: '5rem', color: 'red' }}>{error.toString()}</div>}
-          </>
-        )}
-      </SourceRender.Consumer>
-    </SourceRender>
+    <Provider theme={{ rtl: rtl === 'true' ? true : false }}>
```

Comment:
Not sure whether we should wrap this with div that will have dir='rtl' if rtl is enabled. We have that logic in the ComponentExample component.